### ISSUE TITLE
Backdrop-blur-sm needs backdrop-filter class to be enabled

### ISSUE THREAD

**Issue by moracabanas** [windicss/windicss#396]
In contrast to Tailwind I found I must use both classes at the same time to get the frosted effect applied to some element.

I'm wondering if this is the default behaviour of am I missing something. On Tailwind you can juast say `backdrop-blur-md` and you are good to go

Any thoughts about this?

Thanks you!

**Comment by alexanderniebuhr** [comment]
I am not sure about this one, normally I would say it should work with just `backdrop-blur-md`

**Comment by moracabanas** [comment]
> I am not sure about this one, normally I would say it should work with just `backdrop-blur-md`

Yeah, it should be as in Tailwind but in my case using Windicss I can't get it to work by just `backdrop-blur-md` and don't know why 🤔
